### DISCUSSION TITLE
1.16版本离线账号无法进入服务器

### DISCUSSION THREAD

**Discussion by 1254872** [Meloong-Git/PCL#2018] [提问] [answered]
![屏幕截图 2023-06-10 173318](https://github.com/Hex-Dragon/PCL2/assets/136161268/74347237-b639-4fda-a445-d9317d092cc7)
Minecraft的开发人员表示，1.20会增加正版验证

**Comment by WorldHim** [comment, marked as answer]
见 #1930 常见问题 第二条
> 请断网启动游戏或安装 [Offline Multiplayer](https://github.com/ChickenPige0n/offline-multiplayer-fabric) 或 [MultiOfflineFix](https://github.com/MCTeamPotato/MultiOfflineFix) 模组

**Reply by 1254872** [reply]
可以了，但为什么1.16版本会这样

**Reply by WorldHim** [reply]
目前怀疑是官方更新 1.20 动的手脚，但不确定是无意导致还是有意为之。

**Comment by CYSLK** [comment]
离线进不了服务器的
